### DISCUSSION TITLE
me pueden ayudar con la practica

### DISCUSSION THREAD

**Discussion by edyy-1010** [Launch-X-Latam/MisionFrontEnd#273] [Q&A]
es que no entiendo del todo, los archivos que dejaron de word y el pdf de las primeras semanas alguien me puede orientar

**Comment by GeraldinTorres** [comment]
Todo eso es para conocer a su cliente y asi usted pueda saber que es lo que quiere la persona, conociendo cuales son sus intereses y viendo que tipo de persona es, a que se dedica y esas cosas.

**Comment by edyy-1010** [comment]
que tiempo tenemos para subir todos los trabajos, es que voy quedado

**Reply by Cerecero** [reply]
El proyecto de Abogabot todavia no ay fecha, el de html hasta el 4 de noviembre.
